dev-util/mh/files/Makefile.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/Makefile b/Makefile
+index 01023ca..f87ab56 100644
+--- a/Makefile
++++ b/Makefile
+@@ -14,6 +14,7 @@ endif
+ 
+ DEBUG = -g -fsanitize=address
+ CFLAGS = -Wall -lpcre2-8 #$(DEBUG)
++LDFLAGS ?=
+ OPTS = -D COLOROUTPUT
+ PREFIX = /usr/local
+ MANPATH = /man/man
+@@ -27,7 +28,7 @@ clean:
+ 	rm -fv $(PROGNAME)
+ 
+ mh:  ## compile this software
+-	gcc -o $(PROGNAME) -D VERSION=\"$(VERSION)\" $(OBJECTS) $(CFLAGS) $(OPTS)
++	gcc -o $(PROGNAME) -D VERSION=\"$(VERSION)\" $(OBJECTS) $(CFLAGS) $(LDFLAGS) $(OPTS)
```

Comment:
what if I want to use clang?

Reply:
Fixed, now the makefile uses $(CC) instead of  gcc.